.github/workflows/ci-format.yml

```diff
@@ -52,19 +52,22 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Install SMARTS
+      - name: Install pytest
         run: |
           python3.7 -m venv ${{env.venv_dir}}
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip wheel
-          pip install .[camera-obs,dev,extras,opendrive,ros,test,train]
-      - name: Get changed files
+          pip install pytype==2022.1.13
```

Comment:
Why wont not installing the packages on this runner throw import errors? I remember seeing error for `waymo` packages since we were not installing its dependencies here and was forced to ignore pytype check on files where it was imported.

Reply:
I don't think so, but I'm not 100% sure.  I think what matters most is being in the right "root folder" (in this case, `$GITHUB_WORKSPACE`) such that the `smarts` module references resolve correctly in python.  That is, being in the folder where the repo has been cloned (in the checkout step).  

I tried to test this locally using a tool called `act` before pushing, so I'm fairly confident it won't, but I'm not at 100%.  However, if it does fail on github because of that, we can add that back in.

Having to install all of our dependencies for all of our extensions like `[ros]` and `[waymo]` was exactly what I was trying to avoid.  I'd rather have `pytype` skip checking these (using the "ignore" comments that Adai demonstrated yesterday) than deal with having to diagnose issues that perhaps exist in their interfaces.  However, I don't feel to strongly about that, so I'm willing to add the `pip install .[...]` back into the "install" step if you guys feel it necessary.  

I guess like this, we are probably going to need to add more "ignore" comments to some of our import statements (which I would prefer, longer term, but don't necessarily want to force with this PR), so now I'm thinking I should probably add it back in, at least for the time being.  I guess I'll do that now.

Thanks for asking about this!